iot-e2e-tests/common/pom.xml

```diff
@@ -117,6 +117,29 @@
                     <argLine>
                         -javaagent:${settings.localRepository}/org/jmockit/jmockit/1.24/jmockit-1.24.jar
                     </argLine>
+
+                    <includes>
+                        <!-- Mvn install will run tests found in the files that match the ______Tests.java naming pattern -->
+                        <include>*Tests.java</include>
+                    </includes>
+
+                    <!--
+                    For more details on parallelization in use here, see:
+                    http://maven.apache.org/surefire/maven-surefire-plugin/examples/fork-options-and-parallel-execution.html
+                    -->
+                    <!--Intentionally not using multiple forks. Tests are safe across threads, but don't appear to be safe across jvm instances at the same time-->
+                    <forkCount>1</forkCount>
+
+                    <!--Tests can be run in parallel with tests within their own test class as well as in parallel with tests from other test classes-->
+                    <parallel>both</parallel>
+
+                    <!--Maven will spawn up as many threads as it wants/can while running tests in parallel-->
+                    <useUnlimitedThreads>true</useUnlimitedThreads>
+
+                    <skipITs>${skipITs}</skipITs>
+                    <argLine>
+                        -javaagent:${settings.localRepository}/org/jmockit/jmockit/1.24/jmockit-1.24.jar
+                    </argLine>
```

Comment:
All of these configurations are lifted from the now defunct jvm pom.xml so that the jvm tests still run with the same settings